httpx/client.py

```diff
@@ -237,6 +264,28 @@ def _get_auth_middleware(
 
         return None
 
+    def get_dispatcher_for_url(self, url: URL) -> AsyncDispatcher:
```

Comment:
Also, it seems this method isn't being called anywhere — I assume this is still WIP then?